### DISCUSSION TITLE
No window mode??????????

### DISCUSSION THREAD

**Discussion by JohnVinu** [Stellarium/stellarium#3376] [General]
How to turn on window mod?  Info from there not actual/wrong https://github.com/Stellarium/stellarium/wiki/FAQ#user-content-How_do_I_start_using_Stellarium

**Comment by gzotti** [comment]
Which info is wrong? It says
>Read the Stellarium User Guide

and this has the relevant info in the introductory chapter "A First Tour".

**Reply by JohnVinu** [reply]
![image](https://github.com/Stellarium/stellarium/assets/115151821/e6f8e0eb-de18-4c32-8fae-d7fd4ee8cd8d)
This one. 
 In first, you cant drag screen by any ways; in second, config file named defoult_cfg.ini; in third, any changes there doesnt effect on screen size. 
  So I just want to know is there a ways to use it like normal app with window mode or not?

**Reply by gzotti** [reply]
(Wrong link then. https://github.com/Stellarium/stellarium/wiki/FAQ#how-do-i-change-the-default-window-size)

You NEVER EVER edit default_cfg.ini or any other file in the program directory. 

You press the button which is indicated in "A First Tour" (I don't tell you here, because I want you to read the chapter!)
If you don't find config.ini yet, search the User Guide for it. 

On Windows, after leaving full-screen mode, the window frame is above the visible screen, but you can still grab it. The text in the FAQ may describe X11 behaviour.

**Reply by JohnVinu** [reply]
Sorry, I dont know whe is "A first tour" chapter. I dont use github at all ,and its hard to use to find something...

**Reply by gzotti** [reply]
The Stellarium User Guide is on your harddisk and linked from your start menu. Github is then for issues not solved by reading the User Guide.

**Reply by alex-w** [reply]
F11 or big button on bottom bar

**Reply by JohnVinu** [reply]
Thank you!
